books/free-programming-books-subjects.md

```diff
@@ -314,9 +314,12 @@ Books that cover a specific programming language can be found in the  [BY PROGRA
 * [Learning Modern 3D Graphics Programming](https://web.archive.org/web/20150225192611/http://www.arcsynthesis.org/gltut/index.html) - Jason L. McKesson (draft)
 * [Notes for a Computer Graphics Programming Course](https://www.cs.csustan.edu/~rsc/CS3600F00/Notes.pdf) - Dr. Steve Cunningham (PDF)
 * [OpenGL](http://www.songho.ca/opengl/index.html) - Concepts and illustrations
+* [Physically Based Rendering, Third Edition: from Theory to Implementation](https://www.pbr-book.org/3ed-2018/contents) - Matt Pharr
+* [Ray Tracing Gems Series](https://www.realtimerendering.com/raytracinggems/) - Eric Haines, Adam Marrs
```

Comment:
Good books to have! Don't invent titles! Use the title given by the resource. In this case, create one entry for each of the two books. Note that the two books have different author lists.